app/views/teacher_training_adviser/steps/_subject_interested_teaching.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= f.govuk_collection_select :preferred_teaching_subject_id,
-  GetIntoTeachingApiClient::TypesApi.new.get_teaching_subjects,
-  :id,
-  :value,
+  f.object.class.options,
```

Comment:
Is there a benefit to getting at the class this way versus `SubjectInterestedTeaching.options`?

Reply:
I'd say yes - this means if the class is renamed, than the form wont require changes

Reply:
The only real benefit is that we won't need to change this line if the model name changes so it should make refactoring simpler. I can see an argument that having the class name here may improve readability, but it's a bit six and two threes in my mind.

Reply:
Yeah - makes sense. I was originally thinking of it from the 'where is this used?' point of view; easier to search for it by class. Happy either way though